Argument/Internal/ValidatedNotNullAttribute.cs

```diff
@@ -0,0 +1,8 @@
+using System;
+
+namespace ArgumentInternal {

```

Comment:
Wouldn't the world be better with a dot (I feel a no coming though)

Reply:
Can't do it -- will conflict with `class Argument` which has no namespace.

Reply:
Ah fair enough